docs/source/release_notes.rst

```diff
@@ -8,13 +8,18 @@ Future Release
         * Update to avoid error with a categorical target with unused categories (:pr:`349`)
     * Changes
         * Transition to pure pyproject.toml for project metadata (:pr:`351`)
```

Comment:
I just happened to notice that we still have a `setup.py` hanging around from PR 351. We could delete it here or do it in a separate PR.

Reply:
Removed